src/debugger/osx_ssh.md

```diff
@@ -0,0 +1,16 @@
+# Debugging on MacOS over SSH
+
+If you try to debug a program over SSH, you may experience failures or rizin
+getting stuck while opening the file. This is because the OS is waiting for
+user authentication before allowing the debugging, however since you are over
+SSH the OS has no way of showing the permission window.
+
+To avoid this problem you can either run rizin with `sudo` or you may instruct taskport to not authenticate the user. To do so you can do:
```

Comment:
```suggestion
To avoid this problem you can either run rizin with `sudo` or you may instruct taskport to not authenticate the user by executing the following commands:
```

Reply:
```suggestion
To avoid this problem, you can either run rizin with `sudo` or you may instruct taskport to not authenticate the user. To do so you can do:
```